selenium_webdriver/src/main/java/pageobject/sauceDemoPages/SauceDemoMainPage.java

```diff
@@ -0,0 +1,56 @@
+package pageobject.sauceDemoPages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import pageobject.basePage.BasePage;
+
+import java.util.List;
+
+public class SauceDemoMainPage extends BasePage {
+
+    @FindBy(id = "add-to-cart-sauce-labs-backpack")
+    private WebElement sauceLabsBackPackItemAddToCartButton;
```

Comment:
it works but the approach is not correct. For each item, you to create 'Add to Cart', 'Remove', ... elements

Reply:
Discussed. Should be solved using separate page module with the product fields inside without hardcoding product details like in my case.